src/Backend.hpp

```diff
@@ -34,8 +34,95 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef BACKEND_H_
 #define BACKEND_H_
 
+#include <cmath>
+#include <sstream>
+
+#include "Vector.hpp"
+
 namespace trase {
 
+// pi constant
+const float pi =
+    3.141592653589793238462643383279502884197169399375105820974944592307816406286;
```

Comment:
I think there's enough precision here ;)

Reply:
:) I was wondering if that would even fit in a float!

Reply:
Also, that can presumably be `constexpr` not just `const`?